src/components/NewDraftWizard/WizardComponents/WizardSavedObjectActions.js

```diff
@@ -27,6 +28,7 @@ const useStyles = makeStyles(theme => ({
 }))
 
 type WizardSavedObjectActionsProps = {
+  folderId: string,
```

Comment:
It seems that this `folderId` is not used anywhere in the file?

Reply:
Good call. This prop was used in early version for this feature. Removed on latest commit.